root/event/EditEvent.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * @flow strict-local
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import manifest from '../static/manifest.mjs';
+import EventEditForm
+  from '../static/scripts/event/components/EventEditForm.js';
+
+import EventLayout from './EventLayout.js';
+import type {EventFormT} from './types.js';
+
+component EditEvent(
+  entity: EventT,
+  eventDescriptions: {+[id: string]: string},
+  eventTypes: SelectOptionsT,
+  form: EventFormT,
+) {
+  return (
+    <EventLayout
+      entity={entity}
+      fullWidth
+      page="edit"
+      title={l('Edit event')}
```

Comment:
This would be a new string. But the previous string “Edit” was lacking context for localization.

Reply:
Yeah I think this is a worthy change.

Reply:
I would suggest to keep the old string but to add the context `verb, header`.

Reply:
Why keep it though? I think it's really weird that we have "`Add event`" for creation but just "`Edit`" for editing. The only reason I can see is that we were reusing one file in `entity/edit.tt` so we didn't want to make it complicated, but if we're splitting them, `Edit event` seems a lot better to me. That said, it should still get context.

Reply:
I wouldn't mind keeping "Edit event" since the current page title doesn't mention "event" anywhere.

Reply:
I actually replied before @reosarevok’s reply was visible to me. I don’t mind the addition either, but it is still worth a context for localization.